src/expr/lazy_proof_chain.cpp

```diff
@@ -40,7 +40,7 @@ const std::map<Node, std::shared_ptr<ProofNode>> LazyCDProofChain::getLinks()
     const
 {
   std::map<Node, std::shared_ptr<ProofNode>> links;
-  for (const std::pair<const Node, ProofGenerator*>& link : d_gens)
+  for (const std::pair<const Node, ProofGenerator* const>& link : d_gens)
```

Comment:
Though we have no explicit recommendation, we seem to put `const` before the type everywhere else.

Reply:
`const ProofGenerator*` and `ProofGenerator* const` are different. The latter means that the pointer itself cannot be changed (e.g. to point at a different object) while the former means that the object itself cannot be changed and it is what we seem to be getting from `d_gens` according to the compiler.

Reply:
Ah, indeed. I always get confused about whether `const T*` is equivalent to `T const *` or `T* const`... everything is fine then.